src/tasks/shop-admin/Rule/AssignEntitiesToRule.ts

```diff
@@ -0,0 +1,57 @@
+import {Locator, test as base} from '@playwright/test';
+import type {Task} from '../../../types/Task';
+import type {FixtureTypes} from '../../../types/FixtureTypes';
+
+interface Entity {
+    id: string;
+    name: string;
+    ruleType: string;
+}
```

Comment:
Is there a way to use the actual entity as a parameter? That way, the dev, who wants to use this method, wouldn't need to strip down the entitiy and define the ruleType per entry themself. Instead it would be nice, of the method could identify the ruleType based on the entities, that handed over by the parameter.

Reply:
we have also here a general problem, that the task currently is not providing any information in form of a documentation what the Task need/want as parameter...but yes what kind of ruleTypes are possible is not visible here. We have also other tests where parameters are not 100% sure how to set.